ingresslibchart/templates/_ingress.yaml

```diff
@@ -0,0 +1,26 @@
+{{- define "ingresslibchart.ingress" -}}
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: {{ .Values.name }}-ingress
+  labels:
+    {{- include "helm.labels" . | nindent 4 }}
+  annotations:
+    networking.gke.io/v1beta1.FrontendConfig: "{{ .Chart.Name }}-ingress-frontendconfig"
+    kubernetes.io/ingress.allow-http: "false"
+{{- if not (empty .Values.ingress.cert.preSharedCerts) }}
+    ingress.gcp.kubernetes.io/pre-shared-cert: {{ .Values.ingress.cert.preSharedCerts | join "," | quote }}
+{{- end }}
+    {{- range $key, $value := .Values.ingress.annotations }}
+    {{ $key }}: {{ $value | quote }}
+    {{- end }}
+spec:
+  tls:
+  - secretName: {{ .Chart.Name }}-cert
+  backend:
+    serviceName: {{ .Chart.Name }}-service
```

Comment:
Does the servicePort from the original file need to be provided here or is it okay to omit it?